src/ucp/wireup/wireup.h

```diff
@@ -43,10 +47,10 @@ enum {
 typedef struct {
     const char *title; /* Name of the criteria for debugging */
     uint64_t   local_md_flags; /* Required local MD flags */
-    ucp_wireup_extended_flags_t
-            local_iface_flags; /* Required local interface flags */
-    ucp_wireup_extended_flags_t
-             remote_iface_flags; /* Required remote interface flags */
+    /* Required local interface flags */
+    ucp_wireup_select_flags_t local_iface_flags;
+    /* Required remote interface flags */
+    ucp_wireup_select_flags_t remote_iface_flags;
```

Comment:
format the code of this struct to look like https://github.com/openucx/ucx/blob/master/src/ucp/rndv/proto_rndv.h#L29

Reply:
unresolved

Reply:
reformatted comment above struct and added space line below.